docs/products/clickhouse/howto/sharding.rst

```diff
@@ -0,0 +1,74 @@
+Work with shards in ClickHouse®
```

Comment:
Is it in ClickHouse or Aiven for ClickHouse? These are very different things and should be taken with care to avoid legal issues.